WORKSPACE

```diff
@@ -566,18 +566,21 @@ http_archive(
 # spdlog
 http_archive(
     name = "com_github_gabime_spdlog",
-    url = "https://github.com/gabime/spdlog/archive/v1.4.0.tar.gz",
-    sha256 = "afd18f62d1bc466c60bef088e6b637b0284be88c515cedc59ad4554150af6043",
-    strip_prefix = "spdlog-1.4.0",
+    urls = [ 
```

Comment:
is it related to gitlfs change?

Reply:
When added curl dependency and progress from libgit2 there were compilation problems with new headers. Had to update the SPDLOG and FMT and it fixed the issues.